e2e/Makefile

```diff
@@ -3,54 +3,54 @@ TEST_KUBERNETES_TARGET ?= current
 TEST_SCHEDULER_MANIFEST ?= daemonset
 
 ## Dependency versions
-KIND_VERSION=0.10.0
-MINIKUBE_VERSION=v1.18.1
+KIND_VERSION := 0.10.0
+MINIKUBE_VERSION := v1.18.1
 
-BINDIR := $(shell pwd)/bin
-SUDO=sudo
-KIND_CLUSTER_NAME=topolvm-e2e
+BINDIR := $(PWD)/bin
+SUDO := sudo
```

Comment:
Doing this would prevent overriding variables from command-line like this:
```
$ make SUDO=
```

Revert these changes altogether.